src/EnergyPlus/GroundTemperatureModeling/XingGroundTemperatureModel.cc

```diff
@@ -96,8 +97,9 @@ std::shared_ptr<XingGroundTempsModel> XingGroundTempsModel::XingGTMFactory(Energ
 
             thisModel->objectName = state.dataIPShortCut->cAlphaArgs(1);
             thisModel->objectType = objType;
-            thisModel->groundThermalDiffisivity =
-                state.dataIPShortCut->rNumericArgs(1) / (state.dataIPShortCut->rNumericArgs(2) * state.dataIPShortCut->rNumericArgs(3));
+            thisModel->groundThermalDiffisivity = state.dataIPShortCut->rNumericArgs(1) /
+                                                  (state.dataIPShortCut->rNumericArgs(2) * state.dataIPShortCut->rNumericArgs(3)) *
+                                                  Constant::SecsInDay;
```

Comment:
Yeah, that's going to be a different value :eyes: 

```
groundThermalDiffisivity = rNumericArgs(1) / ( rNumericArgs(2) *rNumericArgs(3) ) * SecsInDay;
```

is:

```math
\alpha_{ground} = \frac{ \frac{W}{m-K} } { \frac{kg}{m^3} * \frac{J}{kg-K} } * \frac{s}{day}
```

```math
\alpha_{ground} = \frac{ m^2 } { s } * \frac{s}{day} = \frac{m^2}{day}
```

This now matches the units required in the model.  :+1: